integration/nwo/standard_networks.go

```diff
@@ -181,6 +181,30 @@ func MultiChannelBasicSolo() *Config {
 	return config
 }
 
+func Ed25519Solo() *Config {
```

Comment:
why do you need a config for ED25519 ? There is nothing unique about this config that is related to ed25519. In the integration test you can just take an existing config, no?

Reply:
I probably don't. I only created there as a temporary solution. Peers can be added externally to this configuration function anyway.